### PR TITLE
GH-597 XYAxis Label Rotation

### PR DESCRIPTION
**Pull request by mlzummo**
@GordonSmith Please Review
@jbrundage 
@dtsnell4 

Adds XYAxis Label Rotation

Fixes GH-597

Signed-off-by: Mathew Zummo <mzummo@gmail.com>

http://rawgit.com/mlzummo/Visualization/XYAXIS_LABEL_ROTATION/demos/dermatology.html?src/chart/Column

### CODE REVIEW

File: src/chart/XYAxis.js [hunk 1]
```diff
@@ -37,6 +38,8 @@
     XYAxis.prototype.publish("yAxisDomainLow", "", "string", "Y-Axis Low");
     XYAxis.prototype.publish("yAxisDomainHigh", "", "string", "Y-Axis High");
     XYAxis.prototype.publish("yAxisDomainPadding", 5, "number", "Y-Axis Low/High Padding (if no low/high specified");
+    XYAxis.prototype.publish("yAxisLabelRotation", null, "number", "Y-Axis Label Rotation");
+
```
Comment (dtsnell4) [inline src/chart/XYAxis.js]: extra space

**Comment by GordonSmith** [comment]
1.  Needs to take orientation into account.
2.  Set x axis rotation to 90deg and its off.
3.  Its messing with the axis labels.

**Comment by mlzummo** [comment]
@GordonSmith do we need rotation of the "YAxis"?

**Comment by GordonSmith** [comment]
Y Axis I don't think so.

**Comment by mlzummo** [comment]
closing in favor of new branch

**Pull request closed without merging**